test/test-apimatic-core/utilities/auth_helper_test.rb

```diff
@@ -52,6 +52,31 @@ def test_is_token_expired_false
     assert_equal expected_value, actual_value
   end
 
+  def test_token_is_expired_with_clock_skew
+    token_expiry = Time.now().utc.to_i + 3600
+    actual_value = AuthHelper.token_expired?(token_expiry, 7200)
+    expected_value = true
+
+    refute_nil actual_value
+    assert_equal expected_value, actual_value
+  end
+
+  def test_token_is_not_expired_with_clock_skew
```

Comment:
Blocker: There should be another test case for a negative clock_skew value so that it helps in avoiding any broken logic in future.